xml/security_fips.xml

```diff
@@ -275,6 +285,40 @@
     initramfs, then reboot.
     </para>
     </sect1>-->
+    <sect1 os="sles" xml:id="sec-how-to-fips-install">
+      <title>Enabling FIPS mode</title>
+      <itemizedlist>
+      <listitem>
+        <para>To enable the kernel FIPS mode which also sets the system policy to FIPS, run:
+         </para>
+<screen>&prompt.user; fips-mode-setup --enable
```

Comment:
Need to mention, that to run "fips-mode-setup --enable" command need to install "crypto-policies-scripts" module using command "zypper install crypto-policies-scripts" or "YAST -> Software -> Software Management". 
Also need to describe functionality of options  "--is-enabled" and "--no-bootcfg".
       •   --is-enabled: Checks the system FIPS mode status and returns failure error code if disabled (2) or inconsistent (1).
       •   --no-bootcfg: The tool will not reconfigure the boot loader, and, instead, will print the options that need to be added to the kernel command line. Exception: it still attempts executing zipl(8) on s390x, as the system might become unbootable otherwise.

Reply:
Instead of the zypper/yast command i would suggest that we just say for "fips-mode-setup" crypto-policies has to be installed. and we recommend that people read the manpage for differences in architectures.

Reply:
Yes good point. Re-phrased. Also the following man pages are referred :
**28.6 More information
For more information, refer to:
Man 8 fips-mode-setup
Man 8 fips-finish-install
Man 7 crypto-policies
Man 8 update-crypto-policies**
Will that suffice?